src/test/java/org/broadinstitute/hellbender/utils/gcs/BucketUtilsTest.java

```diff
@@ -149,7 +150,7 @@ public void testDirSizeGCS() throws IOException, GeneralSecurityException {
         Assert.assertEquals(intermediateFileSize, srcFileSize);
         long intermediateDirSize = BucketUtils.dirSize(intermediate);
         Assert.assertEquals(intermediateDirSize, srcFileSize);
-        long intermediateParentDirSize = BucketUtils.dirSize(BucketUtils.getPathOnGcs(intermediate).getParent().toUri().toString());
+        long intermediateParentDirSize = BucketUtils.dirSize(gcsSubDir);
         Assert.assertEquals(intermediateParentDirSize, srcFileSize);
 
         BucketUtils.deleteFile(intermediate);
```

Comment:
Should this also delete gcsSubDir ?

Reply:
Yes, it should. I'm struggling to run this locally (GCS permissions), which I need to do since the Travis turnaround time is too slow.

Reply:
OK, I have managed to run locally now, and it's working there with this latest fix. It looks like you don't need to explicitly delete gcsSubDir, since when you delete the file in it, the directory disappears (because of the way GCS is not really a filesystem).

Thanks for taking a look @cmnbroad!

Reply:
I tried running it locally and was having the same problem. Temporary revert is always an option in the short term if you're time-bound.

Reply:
I think our messages crossed. Travis is now running the latest fix, so let's see if that passes.